### ISSUE TITLE
Storage disk not displayed in UI after system update

### ISSUE THREAD

**Issue by cotosso** [NethServer/nethsecurity#960]
**Summary**
NethSecurity with a storage (or partition) configured for logs.
After a complete system update logs keep to be correctly saved in the storage and the partitions are healthy. 
However, the storage disk is not visible in the UI as either available or configured.

**Steps to Reproduce**
- Configure a NethSecurity for saving log in a storage partition 
- Update the system to another recent version (e.g. 8.3)
- Verify storage and partitions using system logs and CLI tools.  

   Example output:  
   ```
   Number  Start   End     Size    File system  Name  Flags
   128     17.4kB  262kB   245kB                      bios_grub
   1       262kB   17.0MB  16.8MB  fat16              legacy_boot
   2       17.0MB  332MB   315MB
   3       436MB   128GB   128GB   ext2         ext4
   ```
-  Open the NethSecurity UI and navigate to the storage management section.

**Expected Result**
The disk should appear in the UI (storage section) as a configured storage disk.

**Actual Result**
The disk is not displayed in the UI, neither as a disk available for configuration nor as a configured disk.

**Environment**
NethSecurity 8.3

**Comment by gsanchietti** [comment]
**Test case 1**

- Check the issue is not reproducible

**Test case 2**

- After test case 1, try to remove the storage from the UI
- The operation must not fail

**Test case 2**

- Install a clean machine
- Add a USB stick 
- Configure the stick as an extra storage
- Remove the extra storage
- All operations must not fail

**Comment by stephdl** [comment]
Test 1 : verified
upgrade from 8.3 to testing, the storage drive is available in the UI
Test 2 :  verified
once upgraded to testing, we can remove the storage drive and it again without issue
Test 3 : verified
once upgraded to testing, we can add an usb key, use it and remove it

**Comment by gsanchietti** [comment]
Released on 23.05.5-ns.1.4.0
